release/Magarena/scripts/Pyromancer_s_Assault.groovy

```diff
@@ -0,0 +1,23 @@
+[
+    new OtherSpellIsCastTrigger() {
+        @Override
+        public MagicEvent executeTrigger(final MagicGame game,final MagicPermanent permanent, final MagicCardOnStack cardOnStack) {
+            return (permanent.isController(cardOnStack.getController()) && permanent.getController().getSpellsCast() == 1) ?
+                new MagicEvent(
+                    permanent,
+                    NEG_TARGET_CREATURE_OR_PLAYER,
+                    this,
+                    "Whenever PN cast his or her second spell each turn, SN deals 2 damage to target creature or player\$."
```

Comment:
the trigger conditions should be left out, leaving "SN deals 2 damage to ..."

Reply:
fixed